### PR TITLE
update readme for changes in terminology

### PR DESCRIPTION
**Pull request by avigis**
### Desired Outcome

*Please describe the desired outcome for this PR.  Said another way, what was
the original request that resulted in these code changes?  Feel free to copy
this information from the connected issue.*

### Implemented Changes

*Describe how the desired outcome above has been achieved with this PR. In
particular, consider:*

- _What's changed? Why were these changes made?_
- _How should the reviewer approach this PR, especially if manual tests are required?_
- _Are there relevant screenshots you can add to the PR description?_

### Connected Issue/Story

Resolves #[relevant GitHub issue(s), e.g. 76]

CyberArk internal issue link: [insert issue ID]()

### Definition of Done
*At least 1 todo must be completed in the sections below for the PR to be
merged.*

#### Changelog

- [ ] The CHANGELOG has been updated, or
- [ ] This PR does not include user-facing changes and doesn't require a
  CHANGELOG update

#### Test coverage

- [ ] This PR includes new unit and integration tests to go with the code
  changes, or
- [ ] The changes in this PR do not require tests

#### Documentation

- [ ] Docs (e.g. `README`s) were updated in this PR
- [ ] A follow-up issue to update official docs has been filed here: [insert issue ID]()
- [ ] This PR does not require updating any documentation

#### Behavior

- [ ] This PR changes product behavior and has been reviewed by a PO, or
- [ ] These changes are part of a larger initiative that will be reviewed later, or
- [ ] No behavior was changed with this PR

#### Security

- [ ] Security architect has reviewed the changes in this PR,
- [ ] These changes are part of a larger initiative with a separate security review, or
- [ ] There are no security aspects to these changes

### CODE REVIEW

File: Dual Account Script/README.md [hunk 1]
```diff
@@ -12,14 +12,16 @@ For more information, see [here](https://docs.cyberark.com/Product-Doc/OnlineHel
 
 This script enables fully automatic configuration for the dual account feature.
 
+**_Notice:_** In Privilege Cloud Shared Services environment, only password authentication method is supported.
+
```
Comment (hilagross) [inline Dual Account Script/README.md:16]: only password authentication method is supported
what does it mean? why only this method?

File: Dual Account Script/README.md [hunk 2]
```diff
@@ -12,14 +12,16 @@ For more information, see [here](https://docs.cyberark.com/Product-Doc/OnlineHel
...
 ## Requirements
 - This scripts can run against Vault and PVWA v12.1 and up.
 - The script is used to create and configure new accounts as dual account. It does not configure existing account as dual account.
   - Verify the accounts you wish to configure as dual account does not exist in the Vault you are working with.
 
 ## Limitations
  
-This script supports only platforms with the following properties:
+This script supports only platforms (in PVWA environment) with the following properties:
```
Comment (hilagross) [inline Dual Account Script/README.md:24]:
```suggestion
This script supports only platforms with the following properties, these properties are required in order to connect to PVWA:
```

**Review by hilagross — CHANGES_REQUESTED** [review]

File: Dual Account Script/README.md (hunk 1 above)
Comment (avigis) [reply Dual Account Script/README.md:16]: the only authentication that is supported is password (not email, sms and etc)

File: Dual Account Script/README.md (hunk 2 above)
Comment (avigis) [reply Dual Account Script/README.md:24]: "platform" in this context means platform like UnixSSH or WindowsLocalAdmin.
and the properties belong to the platform.

As you can see here below, the platform is "Unix via SSH Keys" and its properties are:
Address, Username and SSH Private key (which is not supported)

Screen Shot 2022-05-02 at 14 35 58
e

Comment (hilagross) [reply Dual Account Script/README.md:24]: these are my notes from teams:
[25/04/2022 13:05] Hila Gross

- shared service only password is supported
- in limitation- add that this is in order to connect to PVWA itself
- the note and the usage example should indicate that it is Privilege Cloud Shared Services
- everywhere we talk about pCloud specifically need to change to Privilege Cloud Shared Services

from what I understood this should be what we need in order to connect to PVWA (bullet No.2)

**Review by hilagross — APPROVED** [review]

**Review by jeniaSakirko — APPROVED** [review]
LGTM

**Pull request merged**